.github/workflows/build.yml

```diff
@@ -276,10 +276,3 @@ jobs:
     - name: Install upload deps
       run: |
         pip install uritemplate
```

Comment:
This step can also be removed. `uritemplate` was used by `tools/upload_release_files.py`.

Reply:
Thanks, I didn't realize that, and maybe I should delete `tools/upload_release_files.py` too. It does make me think about the mpy-cross actions; they don't get uploaded anywhere, is that right?

Reply:
> It does make me think about the mpy-cross actions; they don't get uploaded anywhere, is that right?

Correct. Jeff proofed them out build-wise, but there hasn't been anything done with them since. My thoughts are here: https://github.com/adafruit/circuitpython/pull/2551#issuecomment-578452197